docs/en/serverless/cases/create-manage-cases.mdx

```diff
@@ -16,7 +16,7 @@ To create a case in your Observability project:
 
 1. In your ((observability)) project, go to **Cases**.
 1. Click **Create case**.
-1. (Optional) If you defined templates, select one to use its default field values. <DocCallOut template="technical_preview" />
+1. (Optional) If you defined <DocLink slug="/serverless/observability/case-settings" section="templates">templates</DocLink>, select one to use its default field values. <DocCallOut template="technical_preview" />
```

Comment:
It looks a little strange to have the full callout here (and it's especially weird to have a technical preview when everything is a technical preview, but I don't have a workaround for that).

You should use the badge instead of the callout here. https://docs.elastic.dev/docs/reference/doc-badge#template-badges

Reply:
Good catch, thanks!